jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by replacing references (such as "$ref") with an
+                        embedded version of the referenced Schema Resource.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource (and the document's root) MUST identify itself
+                        with an absolute URI using the "$id" keyword, and SHOULD make use of the "$schema"
+                        keyword to identify the dialect it is using, in the root of the schema resource.
+                    </t>
+                    <t>
+                        When a reference applicator is dereferenced as part of the bundling process, the Schema Resource
```

Comment:
"reference applicator" is a new phrase. Does it mean the `$ref` keyword?

Reply:
Yes.  Or `$dynamicRef`.  It is defined somewhere in the spec (or was- I definitely used it in a number of places).  Maybe as "by-reference applicator".

Reply:
Yes. It's also called "by-reference applicator". I need to do a terminology consistency run.